src/angular/usermenu/usermenu.scss

```diff
@@ -76,6 +76,10 @@
   }
 }
 
+.sbb-panel hr {
```

Comment:
The additional style definition is not needed. Instead, the `hr` style of the `.sbb-panel` in `_typography.scss` should be overridden (+/- Line 2626).